src/pages/CircleAdminPage/RemoveCircleModal.tsx

```diff
@@ -26,15 +26,18 @@ export const RemoveCircleModal = ({
 
   const navigate = useNavigate();
   const { showError } = useApeSnackbar();
+  const { fetchManifest, unselectCircle } = useApiBase();
 
   const deleteCircleMutation = useMutation(deleteCircle, {
     onMutate: () => {
       setIsLoading(true);
     },
-    onSuccess: () => {
+    onSuccess: async () => {
       queryClient.invalidateQueries(QUERY_KEY_MY_ORGS);
       queryClient.invalidateQueries(QUERY_KEY_MAIN_HEADER);
-      navigate(paths.circles);
+      await navigate(paths.circles);
+      await unselectCircle();
```

Comment:
would have been nice to have a small PR just for adding `unselectCircle`

Reply:
agreed, i'm struggling to figure out how to manage this kind of stuff. I needed this work to do my testing, so I basically wanted to make a new branch and rebase on top of that, but i couldn't figure out how to do that with graphite, and also someone had merged changes into my branch from main which makes rebasing a pain